dev/deps/spark-deps-hadoop-2.7

```diff
@@ -106,7 +106,7 @@ javax.inject-2.4.0-b34.jar
 javax.servlet-api-3.1.0.jar
 javax.ws.rs-api-2.0.1.jar
 javolution-5.5.1.jar
-jaxb-api-2.2.2.jar
+jaxb-api-2.2.11.jar
```

Comment:
Just in case, is this and below `stax-api` required for upgrade?

Reply:
Thank you for review, @MaxGekk . I'll remove that.